src/phoenix/trace/exporter.py

```diff
@@ -8,12 +8,12 @@
 from urllib.parse import urljoin
 
 import requests
-from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
```

Comment:
this swap is going to fail for those people that have it configured as 6006 for HTTP right? I don't think we can blindly swap this. Even below isn't it gonna default to localhost:6006 which is not GRPC...

Reply:
I meant in the code block below it's pulling the PHOENIX_COLLECTOR_ENDPOINT but that assumed HTTP.

Reply:
Let me add a new env var so this is kept the same.

Reply:
Actually, evals are still sent via HTTP. Let me just revert this and punt on it.